simulation_parameters/microbe_stage/organelles.json

```diff
@@ -1,279 +1,279 @@
 ﻿{
   "pilus": {
-    "mpCost": 30,
-    "initialComposition": {
-      "ammonia": 1,
-      "phosphates": 1
+    "MpCost": 30,
+    "InitialComposition": {
+      "Ammonia": 1,
+      "Phosphates": 1
     },
-    "hexes": [
+    "Hexes": [
       {
         "r": 0,
         "q": 0
       }
     ],
-    "processes": {},
-    "components": {
-      "pilus": {}
-    },
-    "shouldScale": false,
-    "prokaryoteChance": 0.5,
-    "chanceToCreate": 0.5,
-    "mass": 0.3,
-    "displayScene": "res://assets/models/organelles/Pilus.tscn",
-    "name": "ORGANELLE_PILUS",
-    "iconPath": "res://assets/textures/gui/bevel/parts/PilusIcon.png",
-    "editorButtonGroup": "External",
-    "editorButtonOrder": 2,
-    "availableUpgrades": {
-      "none": {
-        "isDefault": true,
-        "name": "UPGRADE_NAME_NONE",
-        "description": "UPGRADE_DESCRIPTION_NONE"
+    "Processes": {},
+    "Components": {
+      "Pilus": {}
+    },
+    "ShouldScale": false,
+    "ProkaryoteChance": 0.5,
+    "ChanceToCreate": 0.5,
+    "Mass": 0.3,
+    "DisplayScene": "res://assets/models/organelles/Pilus.tscn",
+    "Name": "ORGANELLE_PILUS",
+    "IconPath": "res://assets/textures/gui/bevel/parts/PilusIcon.png",
+    "EditorButtonGroup": "External",
+    "EditorButtonOrder": 2,
+    "AvailableUpgrades": {
+      "None": {
```

Comment:
Upgrade names are also internal names and will break things if they are capitalized.

Reply:
So all names inside the `AvailableUpgrades` object should return to camelCase?

Reply:
Yes, the keys directly inside the upgrades dictionary are internal keys referred to by exact values in the C# code.